### PR TITLE
Inegrate with swagger

### PR DESCRIPTION
**Pull request by roman-mazur**
Helium generates a swagger spec and a couple of tasks are added to build.gradle to add this spec to the resulting war and also add swagger-ui resources.

### CODE REVIEW

File: src/main/webapp/spec/spec.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
+
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Example API</title>
```
Comment (roman-mazur) [inline src/main/webapp/spec/spec.html:6]: Based on swagger UI index.html with modified paths and default spec URI.

File: build.gradle [hunk 2]
```diff
@@ -108,4 +120,31 @@ helium {
...
+    }
+  }
+}
+
+// Add swagger spec to the war.
+afterEvaluate {
+  war {
+    dependsOn generateSwaggerSpec, unpackSwaggerUi
+    from file("$unpackSwaggerUi.dest/swagger-ui-$swaggerUiVersion/dist"), {
+      into 'spec/ui'
```
Comment (roman-mazur) [inline build.gradle:143]: Adds swagger UI resources to the war.

File: build.gradle [hunk 3]
```diff
@@ -108,4 +120,31 @@ helium {
...
+    }
+    from file("$generateSwaggerSpec.output"), {
+      into 'spec'
+      rename '.+\\.json', 'public.json'
```
Comment (roman-mazur) [inline build.gradle:147]: Adds swagger spec to the war.

File: .travis.yml [hunk 4]
```diff
@@ -23,12 +23,13 @@ git:
 
 install:
 - ./gradlew -s clean assemble check
+- ./gradlew appengineRun > /dev/null &
+- sleep 7 # TODO: Replace with something like a connection ping.
```
Comment (roman-mazur) [inline .travis.yml:27]: Lucky magic number.

**Comment by roman-mazur** [comment]
Open spec/spec.html to see the API docs. Helium still needs to be modified though. But integration works.

**Comment by roman-mazur** [comment]
http://helium-1222.appspot.com/spec/spec.html

File: build.gradle [hunk 5]
```diff
@@ -16,13 +16,25 @@ apply plugin: 'appengine'
...
+    url 'https://github.com'
+    layout 'pattern', {
+      artifact '[organization]/[module]/archive/v[revision](.[ext])'
+    }
+  }
 }
 
+configurations { swaggerUi }
+
+def swaggerUiVersion = '2.1.4'
```
Comment (Burbaka) [inline build.gradle:29]: I guess we should define versions in such way for all other dependencies

**Comment by Burbaka** [comment]
You should add that URL with api description to the README. in the follow up

**Pull request merged**